src/data.js

```diff
@@ -2265,6 +2265,26 @@ module.exports = [
       'UX',
       'Open Source',
     ],
+  },
+      {
+    name: 'Sean Smith',
+    description: 'Developer, Front End Developer, Craft CMS enthusiast, and podcaster',
+    url: 'https://caffeinecreations.ca/uses',
+    twitter: '@cs_sean',
+    emoji: '',
+    country: 'CA',
```

Comment:
Please use the country flag for the time being, rather than the two-letter code.